app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    ls_file = command.split()
+    if ls_file[1] != ls_file[2] and ls_file[0] == "cp":
```

Comment:
U can unpacking in 3 variables in one line
Ex. command, old_file, new_file = command.split() 
where ls_file[0] == command ...